include/git2/deprecated.h

```diff
@@ -89,6 +89,11 @@ GIT_EXTERN(int) git_blob_create_fromstream_commit(
 	git_writestream *stream);
 GIT_EXTERN(int) git_blob_create_frombuffer(
 	git_oid *id, git_repository *repo, const void *buffer, size_t len);
+GIT_EXTERN(int) git_blob_filtered_content(
```

Comment:
Mh. A short pointer would be nice for users: `/* Deprecated in favor of @git_blob_filter */`

Reply:
Fixed.